src/backend/linux_raw/termios/types.rs

```diff
@@ -52,11 +52,18 @@ pub enum Action {
     IOn = linux_raw_sys::general::TCION,
 }
 
-/// `struct termios` for use with [`tcgetattr`].
+/// `struct termios` for use with [`tcgetattr`] and [`tcsetattr`].
 ///
 /// [`tcgetattr`]: crate::termios::tcgetattr
+/// [`tcsetattr`]: crate::termios::tcsetattr
 pub type Termios = linux_raw_sys::general::termios;
```

Comment:
I think we should avoid making `libc` and `linux-raw-sys` public dependencies.

Reply:
Fair enough, but that's neither here nor there: termios is already implemented like that.

Reply:
Mhmm, fair enough, I didn't see that.